nima_core/cli/recall_fast.py

```
@@ -0,0 +1,200 @@
+        
+        memories = []
+        for mf in memory_files:
+            if mf.exists():
+                try:
+                    with open(mf, 'r') as f:
+                        data = json.load(f)
+                        if isinstance(data, list):
+                            memories.extend(data)
+                        elif isinstance(data, dict) and 'memories' in data:
+                            memories.extend(data['memories'])
+                except:
```

Ruff (0.15.0)

[error] Do not use bare `except`

(E722)